docs/Основные блоки и устройство компьютера/Основные внутренние порты.md

```diff
@@ -0,0 +1,49 @@
+# Основные порты в компьютере
+
+В современных компьютерах существует множество портов для подключения различных устройств. 
+В данном материале рассмотрим основные порты, такие как IDE, SATA, PCI и PCIe.
+
+## 1. IDE (Integrated Drive Electronics)
+
+**Описание:** 
```

Comment:
[ch-md-error-emphasis-instead-of-a-heading](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-emphasis-instead-of-a-heading)